src/org/ods/orchestration/util/MROPipelineUtil.groovy

```diff
@@ -339,6 +340,12 @@ class MROPipelineUtil extends PipelineUtil {
                         } else {
                             this.logger.debug("Repo '${repo.id}' is of type ODS Infrastructure as Code Component/Configuration Management. Nothing to do in phase '${name}' for target environment'${targetEnvToken}'.")
                         }
+                    } else if (repo.type?.toLowerCase() == PipelineConfig.REPO_TYPE_ODS_SAAS_SERVICE) {
+                        if (this.project.isPromotionMode && name == PipelinePhases.DEPLOY) {
```

Comment:
@metmajer ... i think none of this is needed... there is no jenkinsfile 
. ?

Reply:
@clemensutschig been thinking the same... but what if account creation, etc. is needed? Don't documents claim that there has to be a Jenkinsfile?

Reply:
If its that way... then this needs to happen in the build phase as well.. similar to ods infra..

Reply:
Wait a minute... if we would support account creation, etc., then we would need to document this in a TIR, I guess -- which we don't want to, right?

Reply:
In a saas case Everything is done by the vendor..  so i vouch to have nothing in there.. so an empty ^if^